Index: src/spotify/SpotifyManager.h
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>//\n// Created by Harry Skerritt on 06/01/2026.\n//\n\n#ifndef SPOTIFYMANAGER_H\n#define SPOTIFYMANAGER_H\n\n\n\nclass SpotifyManager {\n\n};\n\n\n\n#endif //SPOTIFYMANAGER_H\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/spotify/SpotifyManager.h b/src/spotify/SpotifyManager.h
--- a/src/spotify/SpotifyManager.h	(revision 53ca42c4733781617328fe7a4c409e9d314cd234)
+++ b/src/spotify/SpotifyManager.h	(date 1767746959868)
@@ -5,10 +5,13 @@
 #ifndef SPOTIFYMANAGER_H
 #define SPOTIFYMANAGER_H
 
-
+#include <Arduino.h>
+#include <spotify/spotify.hpp>
+#include <ArduinoJson.h>
 
 class SpotifyManager {
 
+
 };
 
 
Index: src/spotify/SpotifyManager.cpp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>//\n// Created by Harry Skerritt on 06/01/2026.\n//\n\n#include \"SpotifyManager.h\"\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/spotify/SpotifyManager.cpp b/src/spotify/SpotifyManager.cpp
--- a/src/spotify/SpotifyManager.cpp	(revision 53ca42c4733781617328fe7a4c409e9d314cd234)
+++ b/src/spotify/SpotifyManager.cpp	(date 1767746959871)
@@ -2,4 +2,12 @@
 // Created by Harry Skerritt on 06/01/2026.
 //
 
-#include "SpotifyManager.h"
+#include "spotifyManager.h"
+#include <LittleFS.h>
+#include <WiFi.h>
+
+#include "global_state.h"
+
+
+
+
